{"name": "test_fallback_to_primary_server_after_failover[chromium]", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Page.goto: Timeout 30000ms exceeded.\nCall log:\nnavigating to \"http://10.189.200.4/stories\", waiting until \"load\"", "trace": "E   playwright._impl._errors.TimeoutError: Page.goto: Timeout 30000ms exceeded.\n    Call log:\n    navigating to \"http://10.189.200.4/stories\", waiting until \"load\""}, "attachments": [{"name": "log", "source": "270ebc46-fc8b-4cb6-8bd5-66641af10551-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "dc66ae86-14a1-4002-9bc7-156223469d58-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "e251a07f-b381-4de8-85fc-5bfc22204984-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "browser_name", "value": "'chromium'"}], "start": 1757863543721, "stop": 1757863968091, "uuid": "dae716f1-17e7-4a6a-81c8-aebfa9223e15", "historyId": "5b9cfb6c1b4c1fdae083bf762a092068", "testCaseId": "f60cc0c46287a623eb80e98db1d7b24f", "fullName": "tests.Automation_tests.test_VFD#test_fallback_to_primary_server_after_failover", "labels": [{"name": "feature", "value": "File Distribution"}, {"name": "story", "value": "Fallback to Primary Server After Failover"}, {"name": "parentSuite", "value": "tests.Automation_tests"}, {"name": "suite", "value": "test_VFD"}, {"name": "host", "value": "TR-GKJF6LPM4P"}, {"name": "thread", "value": "8448-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.Automation_tests.test_VFD"}]}